schemas/styles/csl-variables.rnc

```diff
@@ -5,7 +5,12 @@ namespace a = "http://relaxng.org/ns/compatibility/annotations/1.0"
 div {
   
   ## All variables
-  variables = variables.dates | variables.names | variables.standard
+  variables =
+    variables.dates
+    | variables.names
+    | variables.numbers
+    | variables.titles
+    | variables.strings
   
   ## Standard variables
   variables.standard = variables.numbers | variables.strings
```

Comment:
not aimed at @denismaier, but why do we have this pattern if we don't use it? the "variables" pattern above could, for example

Reply:
basically, my suggestion is either we remove it here, or we use it above. either way, I can make the change directly.

Reply:
`variables.standard` is what can be rendered with `cs:text`. It should be restored above under `variables` and `variables.title` should be added to `variables.standard`.

Reply:
Good catch @bwiernik. 

In that case, please make that change @denismaier, and make sure you run trang again before you commit it (because I think it will change the formatting of that pattern).

Reply:
Thanks for catching this. Will deal with it tomorrow.

Reply:
Done.